css/style.css

```diff
@@ -116,7 +116,9 @@ ul.nav {
 
 a, a:hover, a:focus {
   color: #6eceb2;
-  font-weight: bold;
+}
+a.inline-link {
```

Comment:
Not sure this is where I want this yet. It's used on the about page for a link within a paragraph. The yellow and aqua colors just aren't providing enough contrast on white background. Should we go back to a dark site background or do we like the white?